### PR TITLE
Tweaks to the docs

### PR DESCRIPTION
**Pull request by tomwilkie**
- fix up formatting for new k8s instructions
- remove old k8s instructions

### CODE REVIEW

File: site/installing.md [hunk 1]
```diff
@@ -173,26 +173,18 @@ It is recommended that you run Scope natively in your Kubernetes cluster using t
...
 `LoadBalancer` or `NodePort`, by default this is unspecified (only internal access)
 
 To download the Scope manifest, use the YAML version of the manifest, since it is easier to read, for example:
 
-```
-curl --silent --remote-name https://scope.weave.works/k8s-gen/weavescope.yaml
-```
-
-    kubectl create -f https://git.io/scope-k8s
+    curl --silent --remote-name https://scope.weave.works/k8s-gen/weavescope.yaml
```
Comment (tomwilkie) [inline site/installing.md]: @ilya this doesn't seem to work:

```
Toms-MacBook-Pro:example twilkie$ curl https://scope.weave.works/k8s-gen/weavescope.yaml
<html>
<head><title>404 Not Found</title></head>
<body bgcolor="white">
<center><h1>404 Not Found</h1></center>
<hr><center>nginx/1.9.11</center>
</body>
</html>
```

File: site/installing.md [hunk 2]
```diff
@@ -173,26 +173,18 @@ It is recommended that you run Scope natively in your Kubernetes cluster using t
 
 The simplest way to get the latest release of Scope deployed onto a Kubernetes cluster is the following:
 
-```
-kubectl create -f 'https://scope.weave.works/k8s-gen/weavescope.json' --validate=false
-```
+    kubectl create -f 'https://scope.weave.works/k8s-gen/weavescope.json' --validate=false
```
Comment (tomwilkie) [inline site/installing.md]: @errordeveloper why is the `--validate=false` necessary?  Its kinda ugly.

File: site/installing.md (hunk 1 above)
Comment (errordeveloper) [inline site/installing.md]: Yeah, waiting on service PR.

File: site/installing.md (hunk 2 above)
Comment (errordeveloper) [inline site/installing.md]: Yes, I'm afraid it's currently needed. Due to a bug in Kubernetes, but we can figure it out later, if it really bothers people.

**Comment by errordeveloper** [comment]
LGTM

File: site/installing.md (hunk 1 above)
Comment (tomwilkie) [inline site/installing.md]: Who's reviewing the service PR?  We should remove the instructions from the scope docs (reverting to previous instructions) until we get this sorted.

File: site/installing.md (hunk 2 above)
Comment (tomwilkie) [inline site/installing.md]: Whats the bug?  We should link to it from here.

Comment (errordeveloper) [inline site/installing.md]: #1474

**Pull request merged**